.gitignore

```diff
@@ -0,0 +1,7 @@
+# Ignore Python bytecode files
+__pycache__/
+
+# Ignore static files
+static/
+
+migrations/
```

Comment:
Migrations will not be added to gitignore unless asked. In some cases we put them in gitignore and in some we do not.